### PR TITLE
Rename instances of browserid to bidpom

### PR DESCRIPTION
**Pull request by bobsilverberg**
I will be releasing this to PyPI as `bidpom` version 2.0

@davehunt r?

### CODE REVIEW

File: tests/check_add_email.py [hunk 1]
```diff
@@ -16,14 +16,14 @@
 class TestAddEmail(BaseTest):
 
     def test_add_email(self, base_url, selenium, timeout, verified_user, new_email):
-        browser_id = BrowserID(selenium, timeout)
-        browser_id.sign_in(verified_user['email'], verified_user['password'])
+        bidpom = BIDPOM(selenium, timeout)
+        bidpom.sign_in(verified_user['email'], verified_user['password'])
```
Comment (davehunt) [inline tests/check_add_email.py]: Despite my original reaction, I actually think this is fine. In tests we could choose to name this `browser_id` or even `persona` for improved readability, but it's not something we need to do here.

**Comment by davehunt** [comment]
r+

**Comment by bobsilverberg** [comment]
Thanks @davehunt. I realized that the documentation needs to be updated too, and when I went to look at the readme I found that the docs actually live in the GitHub wiki. I guess we should probably change that as it's non-standard. I will still merge this, but what do you think we should do about the docs?

**Comment by davehunt** [comment]
> Thanks @davehunt. I realized that the documentation needs to be updated too, and when I went to look at the readme I found that the docs actually live in the GitHub wiki. I guess we should probably change that as it's non-standard. I will still merge this, but what do you think we should do about the docs? 

Let's just update the wiki.

**Comment by bobsilverberg** [comment]
SGTM

**Pull request merged**